src/v2/Apps/Conversation/Components/Details.tsx

```diff
@@ -80,6 +110,19 @@ export const Details: FC<DetailsProps> = ({ conversation, ...props }) => {
     )
   })
 
+  const getDetailsContainerWidth = () => {
+    // For big screens
+    if (width > parseInt(breakpoints.xs, 10)) {
+      if (showDetails) {
+        return "376px"
+      }
+      return ["376px", "376px", "376px", "376px", "0"]
+    }
+
+    // For small screens
+    return "100%"
```

Comment:
Cool 💯  It looks much better to me but good to double-check with Will (if you haven't done already) if detail box has to be full width on mobile.